eval/src/vespa/eval/eval/cell_type.cpp

```diff
@@ -33,7 +33,7 @@ CellTypeUtils::mem_size(CellType cell_type, size_t sz)
 std::vector<CellType>
 CellTypeUtils::list_types()
 {
-    return {CellType::FLOAT, CellType::DOUBLE};
+    return {CellType::INT8, CellType::BFLOAT16, CellType::FLOAT, CellType::DOUBLE};
```

Comment:
I suggest listing types in their natural order (same as enum). Also consider adding separate utility functions for stable/unstable cell types.